Source/UI/CommonUI/UIStyle/CKOUIStyle+UIFont.swift

```diff
@@ -1,55 +1,55 @@
 import UIKit
 
 /// Default Checkout UI Style
-public enum CKOUIStyle {}
```

Comment:
Files and signatures will also need renaming.

But I'm not sure how I feel about `UI...` since this can show up under standard UIKit naming domain (UIView, UIColor, UIScene, UIImage....). I would rather we clearly set it apart from Apple API.

How would you feel about `PaymentStyle`, `FramesStyle`, `DefaultUIStyle`... or other more appropriate name you can think of?